### PR TITLE
Add competition info, and remove code competitions

### PR DESCRIPTION
**Pull request by juniwbjerde**
Under the competitions page I've added a tab for showing info about competitions, and removed the tab for code competitions (as the jubilee board decided not to host code competitions)

### CODE REVIEW

**Comment by juniwbjerde** [comment]
As there's no breaking changes here and this should be ready for tomorrow, i'll merge it now. If anyone has any input on it I'll change it later on:)

File: components/CompetitionInfo.tsx [hunk 1]
```diff
@@ -0,0 +1,117 @@
+import { Card, CardContent, Typography } from "@mui/material";
+import React from "react";
+
+const CompetitionInfo = () => {
+  return (
+    <Card sx={{ mt: 7, borderTop: "3px solid", borderColor: "primary.main" }}>
```
Comment (eikhr) [inline components/CompetitionInfo.tsx:6]:
```suggestion
    <Card
      sx={{
        mt: 7,
        borderTop: "3px solid",
        borderColor: "primary.main",
        maxWidth: "1200px",
      }}
    >
```

Add a max width, on large screens it gets too wide
Screenshot 2022-03-07 at 08 11 24

Comment (juniwbjerde) [reply components/CompetitionInfo.tsx:6]: Totally agree that this is needed!

Comment (juniwbjerde) [reply components/CompetitionInfo.tsx:6]: Fixed in #95 :)

**Pull request merged**